server/test/unit/com/thoughtworks/go/server/presentation/models/JobDetailPresentationModelJMockTest.java

```diff
@@ -80,119 +75,6 @@ public void tearDown() throws Exception {
     }
 
     @Test
-    public void hasBuildErrorShouldBeFalseWhenEmptyContent() throws Exception {
-        context.checking(new Expectations() {
-            {
-                allowing(stubJobInstance).getBuildError();
-                will(returnValue(""));
-            }
-        });
-        assertThat(jobDetailPresenter.hasBuildError(), is(false));
-
-    }
-
-    @Test
```

Comment:
@jyotisingh — I hope I'm not missing something. But I can't seem to be able to find references to these methods. Can you recheck?